wvd-templates/Create and provision WVD host pool/mainTemplate.json

```diff
@@ -98,6 +98,13 @@
                 "description": "True indicating you would like to use managed disks or false indicating you would like to use unmanaged disks."
             }
         },
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "The Name of the Storage Account(Unique Name) to store unamanged disks for unmanaged gallery VM"
```

Comment:
Change this to be:
"description": "(Required when rdshImageSource = Gallery and RdshUseManagedDisks = False) The name of the storage account to store the unmanaged disks from an Azure Gallery image. If you decide to use unmanaged disks from a custom vhd, the disks will be stored in the same storage account as the image."

Also, please update the ReadMe with the same description.